README.md

```diff
@@ -161,6 +161,88 @@ EventTracer.info action: 'Action', message: 'Message',
 #  counter_2, 2
 ```
 
+
+### DynamoDB integration
+
+**Prerequisites:**
+- Sidekiq
+- AWS DynamoDB SDK
+
+Before using this logger, define the app name that will be sent with each log to DynamoDB:
+```ruby
+EventTracer::APP_NAME = 'guardhouse'.freeze
+```
+
+**Preparing payload (optional)**
+If you have any pre-processing to be done, you can define a private method `prepare_payload` in `EventTracer::DynamoDBLogger`
+
+```ruby
+module EventTracer
+  class DynamoDBLogger
+
+    private
+
+      def prepare_payload(log_type, action:, message:, args:)
+        ...
+
+        args.merge(
+          event: event,
+          timestamp: Time.now.utc.iso8601(6),
+          action: action,
+          message: message,
+          log_type: log_type,
+          app: 'mission_control'
+        )
+      end
+
+  end
+end
+```
+
+Alternatively, you can choose to override the `save_message` method:
+
+```ruby
+module EventTracer
+  class DynamoDBLogger
+
+    private
+
+      def save_message(log_type, action:, message:, **args)
+        ...
+
+        payload = args.merge(
+          event: event,
+          timestamp: Time.now.utc.iso8601(6),
+          action: action,
+          message: message,
+          log_type: log_type,
+          app: 'mission_control'
+        )
+
+        DynamoDBLogWorker.perform_async(payload)
+        LogResult.new(true)
+      end
+
+  end
+end
+```
+
+
+**Buffer for network/IO optimization (optional)**
+
+For this logger, a thread-safe buffer has been implemented to allow batch sending of logs. To utilise the buffer, define a buffer with an optional keyword argument `buffer_size`, as such:
+```ruby
+buffer = EventTracer::Buffer.new(
+  buffer_size: ENV.fetch('EVENT_TRACER_DEFAULT_BUFFER_SIZE', EventTracer::Buffer::DEFAULT_BUFFER_SIZE).to_i
```

Comment:
do u want to expose the buffer interval for config too?

Reply:
Hm... true, we could.

Reply:
Realised I just needed to document it. Already exposed.